apps-rendering/src/components/shared/kicker.tsx

```diff
@@ -0,0 +1,93 @@
+import type { SerializedStyles } from '@emotion/react';
+import { css } from '@emotion/react';
+import { text as kickerText } from '@guardian/common-rendering/src/editorialPalette';
+import type { ArticleFormat } from '@guardian/libs';
+import { ArticleDesign } from '@guardian/libs';
+import { remSpace } from '@guardian/source-foundations';
+import { OptionKind } from '@guardian/types';
+import type { Option } from '@guardian/types';
+import type { ReactElement } from 'react';
+
+const dotStyles = (colour: string): SerializedStyles => {
+	return css`
+		color: ${colour};
+		:before {
+			border-radius: 62.5rem;
+			display: inline-block;
+			position: relative;
+			background-color: currentColor;
+			width: 0.75em;
+			height: 0.75em;
+			content: '';
+			margin-right: ${remSpace[1]};
+			vertical-align: initial;
+		}
+	`;
+};
+
+const liveDot = (
+	format: ArticleFormat,
+	colour: string,
+): ReactElement | null => {
+	if (format.design === ArticleDesign.LiveBlog) {
+		return <span css={dotStyles(colour)} />;
+	} else {
+		return null;
+	}
+};
```

Comment:
This could technically be considered a React component in its own right?

```suggestion
type LiveDotProps = {
  format: ArticleFormat,
  colour: string,
}

const LiveDot: FC<LiveDotProps> = ({ format, colour }) => {
	if (format.design === ArticleDesign.LiveBlog) {
		return <span css={dotStyles(colour)} />;
	} else {
		return null;
	}
};
```

Which is then rendered like this:

```tsx
<LiveDot format={format} colour={kickerColour} />
```

I don't think it's necessary to re-write it btw, just an FYI.